vignettes/data_analysis_single_dose_treatment_workflow.Rmd

```diff
@@ -107,7 +107,7 @@ Next, we log2 transform our intensities, then normalise the data to the median v
 
 Note that we are also using the pipe operator `%>%` included in the R package [`magrittr`](https://magrittr.tidyverse.org/index.html). `%>%` takes the output of the preceding function and supplies it as the first argument of the following function. Using `%>%` makes code easier to read and follow.
 
-For median normalisation we are using the **protti** function `median_normalisation()`. The function normalises intensities for each run to the median of all runs. This is only necessary if your search algorithm does not already median normalise your intensities. For the example data we have disabled median normalisation in Spectronaut, therefore we need to median normalise now. 
+For normalisation we are using the **protti** function `normalise()`. We generally recommend using median normalisation (`method = "median"`). The function normalises intensities for each run to the median of all runs. This is only necessary if your search algorithm does not already median normalise your intensities. For the example data we have disabled median normalisation in Spectronaut, therefore we need to median normalise now. 
```

Comment:
Not sure if I'd say that we recommend to use median normalisation. For some applications there's probably better options. I'd maybe say something along the lines of "For this example we will use median normalisation".